pkg/daemon/ceph/osd/volume.go

```diff
@@ -141,6 +141,7 @@ func updateLVMConfig(context *clusterd.Context) error {
 	}
 
 	output := bytes.Replace(input, []byte("udev_sync = 1"), []byte("udev_sync = 0"), 1)
+	output = bytes.Replace(output, []byte("allow_changes_with_duplicate_pvs = 0"), []byte("allow_changes_with_duplicate_pvs = 1"), 1)
```

Comment:
Since this is for rhel8, would this change be affecting the other environments ?

Reply:
The works with the upstream image ceph image as well.  ceph/ceph:v14.2.4-20190917.

Reply:
In centos7 that setting is already set to 1, so this change effectively restores the setting